MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -12,21 +12,31 @@ internal class MiniAppWebView: WKWebView {
         return config
     }
 
-    convenience init(miniAppId: String, versionId: String) {
+    convenience init(miniAppId: String, versionId: String, queryParams: String? = nil) {
         let schemeName = Constants.miniAppSchemePrefix + miniAppId
-        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName)!)
+        let urlRequest = URLRequest(url: URL(string: schemeName + "://miniapp/" + Constants.rootFileName + MiniAppWebView.getQueryParams(queryParams: queryParams))!)
```

Comment:
I've always thought we should not build the URL with concatenation. What do you think of a method executing this code:

```
        guard var urlComps = URLComponents(string: "miniapp/" + Constants.rootFileName) else {
            return nil
        }
        urlComps.scheme = schemeName
        urlComps.queryItems = "queryParams".components(separatedBy: "&").compactMap({ (param) -> URLQueryItem? in
            var components = param.components(separatedBy: "=")
            let name = components[0]
            let value: String
            if components.count > 1 {
                components.remove(at: 0)
                value = components.joined(separator: "=")
            }else {
                value = ""
            }
            return URLQueryItem(name: name, value: value)
        })
        
        guard let url = urlComps.url else {
            return nil
        }
        
        return URLRequest(url:url)
```

Reply:
As mentioned below, I think we should make it the mini app's responsibility to parse the params. We should only care about two cases in the params:
- If the params can somehow break the loading of the mini app (prevent it from loading)
- If there is some security vulnerability with allowing certain types of params